aspnetcore/signalr/redis-backplane.md

```diff
@@ -181,7 +181,7 @@ services.AddSignalR()
 
 ## Clustering
 
-Clustering is a method for achieving high availability by using multiple Redis servers. Clustering isn't officially supported, but it might work.
+[Reddis Clustering](https://redis.io/topics/cluster-spec) is a method for achieving high availability by using multiple Redis servers. Clustering isn't officially supported, but it might work.
```

Comment:
```suggestion
[Redis Clustering](https://redis.io/topics/cluster-spec) is a method for achieving high availability by using multiple Redis servers. Clustering isn't officially supported, but it might work.
```